dotcom-rendering/src/lib/adStyles.ts

```diff
@@ -165,18 +165,16 @@ const spacefinderAdSlotStyles = css`
 	/* Give ad slots inserted on the client side a placeholder height.
    Let the ad slot take control of its height once rendered. */
 	.ad-slot--inline:not(.ad-slot--rendered) {
-		min-height: ${adSizes.outstreamMobile.height +
-		constants.AD_LABEL_HEIGHT}px;
-
-		${from.desktop} {
-			min-height: ${adSizes.mpu.height + constants.AD_LABEL_HEIGHT}px;
-		}
+		min-height: ${adSizes.mpu.height + constants.AD_LABEL_HEIGHT}px;
 	}
 
 	/* Give inline1 ad slot a different placeholder height comparing to subsequent-inlines to reduce CLS.
    Let the ad slot take control of its height once rendered.
    IMPORTANT NOTE: We currently do not serve OPT-OUT for inline1 but we will need to change this value before we do. */
 	.ad-slot--inline1:not(.ad-slot--rendered) {
```

Comment:
Maybe it'd be nicer to reserve MPU height for inline1 and vertically center the outstream ad, like we do for MPUs in the Teads slot on desktop?

Reply:
Yep good idea, done! I've also added a background colour to the slot because the vertical centering looked off without it